src/components/__tests__/Workspace.test.js

```diff
@@ -19,10 +19,12 @@ import { COVERED, NOT_COVERED } from '@/actions/sensor';
 import { pushCommand } from '@/actions/rover';
 
 jest.mock('node-blockly/browser');
+jest.mock('sumo-logger');
```

Comment:
There's a remix test that you could add an assertion to that `log` was called like the logger test does